src/views/officer_seattle.j2

```diff
@@ -8,4 +8,4 @@
 <p><b>Hourly Rate:</b> ${{ hourly_rate }}</p>
 <p><b>Projected salary (w/o overtime):</b> ${{ projected }}</p>
 {% endif %}
-<p><i>*data last updated 1/28/21</i></p>
+<p><i>Record Source Roster Date: {{ date }}</i></p>
```

Comment:
You're removing the asterisk from the start of this, should we remove it elsewhere? Or only add it to the salary fields here and move it into the if block?

Reply:
Oh yeah I did this before I knew what it was for. I'll add an `*` to the thing.